models/articles.models.js

```diff
@@ -14,24 +14,39 @@ function selectArticleById(article_id) {
       }
     })
 }
-function selectAllArticles() {
+function selectAllArticles(sort_by = "created_at", order_by = "DESC") {
+  const validSortBy = [
+    "article_id",
+    "title",
+    "topic",
+    "author",
+    "created_at",
+    "votes",
+    "comment_count",
+  ]
```

Comment:
You have sort bys here that aren't tested. Think about the column and whether a real user will know what they mean.